concepts/loops/introduction.md

```diff
@@ -0,0 +1,3 @@
+# Introduction
+
+Loops enable repetition of blocks of code. There are three types of loops in Rust: the `for` loop that enables looping over elements of a collection, the `while` loop that repeates a block of code as long as a condition is satisfied and the `loop` loop that indefinitely repeats code until a `break` is invoked explicitly.
```

Comment:
Same here: `for` loops fundamentally act on anything implementing `IntoIterator`. Collections have that property, but so do iterators and certain other special-case types.